deploy/packer/download-datacap-nightly.sh

```
@@ -0,0 +1,43 @@
+  if ! command -v "$command" >/dev/null 2>&1; then
+    echo "missing required command '$command'; $install_hint" >&2
+    exit 1
+  fi
+}
+
+require_command gh "install GitHub CLI: https://cli.github.com/"
+require_command dpkg-deb "install the dpkg package (for example: sudo apt-get install dpkg)"
+
+readonly output_directory="$1"
+readonly repository="getlantern/lantern-box"
+readonly package_directory="$(mktemp -d)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)